src/main/scala/org/globalforestwatch/summarystats/SummaryMain.scala

```diff
@@ -217,25 +229,12 @@ object SummaryMain
               SummarySparkSession("GFW Summary Stats Spark Session")
 //            import spark.implicits._
 
-            // ref: https://github.com/databricks/spark-csv
-            val featuresDF: DataFrame =
-              FeatureDF(featureUris, featureObj, kwargs, spark)
-
-            /* Transition from DataFrame to RDD in order to work with GeoTrellis features */
-            val featureRDD: RDD[Feature[Geometry, FeatureId]] =
-              FeatureRDD(featuresDF, featureObj)
-
-            val inputPartitionMultiplier = 64
-
-            val part = new HashPartitioner(
-              partitions = featureRDD.getNumPartitions * inputPartitionMultiplier
-            )
 
-            SummaryAnalysisFactory(
+             SummaryAnalysisFactory(
               analysis,
-              featureRDD,
+              featureObj,
```

Comment:
See comments above. Maybe a good opportunity to add a featureRDD factory before this line. This way, we the spatial RDD can be created upfront and may be used in other future models as well